src/uu/tr/src/tr.rs

```diff
@@ -126,6 +126,13 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         translating,
     )?;
 
+    if is_stdin_directory(&stdin) {
+        return Err(USimpleError::new(
+            1,
+            format!("{}: read error: Is a directory", uucore::util_name()),
```

Comment:
`format!` is not necessary because `USimpleError` already prefixes the message with the util name.